.github/workflows/release.yml

```diff
@@ -0,0 +1,46 @@
+name: release
+
+on:
+  release:
+    types: [published]
+
+jobs:
+  build:
+    strategy:
+      fail-fast: true
+      matrix:
+        node:
+          - 18
+        platform:
+          - ubuntu-latest
+    name: "${{matrix.platform}} / Node.js ${{ matrix.node }}"
+    runs-on: ${{matrix.platform}}
+
+    steps:
+      - uses: actions/checkout@v4
+        with:
+          fetch-depth: 0
```

Comment:
Might need to specify which code to check out e.g. ```ref: ${{ github.event.release.tagName }}``` ([doc](https://docs.github.com/en/graphql/reference/objects#release))

Reply:
I think its OK as is from reading the actions/checkout docs $GITHUB_SHA should be set to the last commit in the tagged release. If you've got prior experience that says this isn't true, I can add `ref` as well. I haven't dealt with Github actions that exensively.

https://github.com/actions/checkout
https://docs.github.com/en/actions/using-workflows/events-that-trigger-workflows#release

Reply:
You are correct. Better left as is.